airbyte-integrations/connectors/source-orb/bootstrap.md

```diff
@@ -6,10 +6,11 @@ Orb is a REST API. Connector has the following streams, and all of them support
 * [Plans](https://docs.withorb.com/reference/list-plans)
 * [Customers](https://docs.withorb.com/reference/list-customers) 
 * [Credits Ledger Entries](https://docs.withorb.com/reference/view-credits-ledger)
+* [Invoices](https://docs.withorb.com/docs/orb-docs/api-reference/schemas/invoice)
 
 Note that the Credits Ledger Entries must read all Customers for an incremental sync, but will only incrementally return new ledger entries for each customer.
 
-Since the Orb API does not allow querying objects based on `updated_at`, these incremental syncs will capture updates to newly created objects but not resources updated after object creation.
+Since the Orb API does not allow querying objects based on `updated_at`, these incremental syncs will capture updates to newly created objects but not resources updated after object creation. Use a full resync in order to capture newly updated entries.
```

Comment:
is the incremental sync mode still useful if it doesn't update existing records? this seems like a non-obvious gotcha

Reply:
Ah, I didn't modify this behavior (just clarified). I believe it's still useful to have an incremental sync if you're only interested in capturing _newly created resources_ and not state transitions. This is consistent with (for example) the [Stripe Connector](https://docs.airbyte.com/integrations/sources/stripe/#supported-sync-modes) since Stripe also does not expose an update time for its resources.

(For your context, there's a larger refactor coming here where we'll support a more useful version of incremental sync by paginating through resource events  on the resources themselves to detect newly changed resources)